crowsnest/logging_helper.py

```
@@ -0,0 +1,175 @@
+    else:
+        logger.log_info_silent(f"{cam.path}")
+    logger.log_info_silent(f"Supported Formats:")
+    log_camera_formats(cam)
+    logger.log_info_silent(f"Supported Controls:")
+    log_camera_ctrls(cam)
+
+
+def log_legacy_cam(camera_path: str) -> None:
+    cam: camera.UVC = camera.camera_manager.get_cam_by_path(camera_path)
+    logger.log_info(f"Detected 'Raspicam' Device -> {camera_path}")
+    logger.log_info_silent(f"Supported Formats:")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)